e2e/rbd.go

```diff
@@ -68,15 +81,113 @@ var _ = Describe("RBD", func() {
 			}
 
 			By("create a PVC and Bind it to an app", func() {
```

Comment:
The narrative here creates the PVC twice once for normal PVC testing and once for the snapshot testing (and later a different one for 2 PVC/App create test). A shorter but similar end result narrative for the tests would be to use the PVC created in the step here for the snapshot test, and later create an additional one for the 2 PVC test.

Something to consider in the **future** if the CI time is longer, would be to create the above narrative to be more efficient in testing the flow.

NOTE: As per my understanding Ginkgo runs the BeforeEach and AfterEach for each `It` block and not each `By` block, if it is the latter then the above narrative is not feasible.